app/controllers/concerns/idv_session.rb

```diff
@@ -26,16 +28,23 @@ def idv_session
   protected
 
   def idv_attempts=(num)
-    user_session[:idv_attempts] = num
+    current_user.update!(idv_attempts: num)
   end
 
   def idv_attempts
-    user_session[:idv_attempts] ||= 0
+    current_user.idv_attempts ||= 0
+  end
```

Comment:
I thought of that. I do like the encapsulation of hiding the storage details of the IdV related values in the IdvSession concern, which meant in this case that I didn't need to change any controller code when I changed the storage implementation.